tools/internal_ci/linux/arm64/grpc_bazel_test_c_cpp_in_docker.sh

```diff
@@ -25,5 +25,13 @@ cd /var/local/git/grpc
 # tests require port server to be running
 python3 tools/run_tests/start_port_server.py
 
+python3 tools/run_tests/python_utils/bazel_report_helper.py --report_path bazel_test_c_cpp
+
 # test gRPC C/C++ with bazel
```

Comment:
Since the bazel wrapper is generated for a single use, I would suggest to put line 28 and line 31 together and move the comment up. This sends a better signal to readers that these two commands are coupled and are expected to be called together.

Reply:
done.